### ISSUE TITLE
⚠️ What is diff between this plugin and https://github.com/felixfbecker/semantic-release-docker

### ISSUE THREAD

**Issue by tomavic** [esatterwhite/semantic-release-docker#33]
What is diff between this plugin and https://github.com/felixfbecker/semantic-release-docker

I can see that your plugin is much more updated and maintained. This should have more stars than the other plugin so that people coming from Google search like me don't be confused.

Also, please try to add your plugin here as well https://github.com/semantic-release/semantic-release/blob/master/docs/extending/plugins-list.md

**Comment by esatterwhite** [comment]
@tomavic I haven't used this one specifically, But I would have to say, mine is much more configurable and has more template hooks for you to get the most out of it.  Additionally, I found the other plugins quite a challenge to work with when operating a mono repo and needing to publish multiple docker images. It plays nice with [multi-semantic-release] https://github.com/dhoulb/multi-semantic-release 

This was my primary focus when building this plugin. easy to use by default, configurable at (almost) every level, and works pretty well in a mono repo

**Comment by tomavic** [comment]
Can I use this plugin to create a custom docker container that will be used to run my commands (npm i, npm run build, npx semantic-release)

**Comment by esatterwhite** [comment]
you want this plugin to run npm commands, or you want npm commands to use this plugin to build a container?

**Comment by tomavic** [comment]
Use it to build my container, then run some npm commands from inside this container.

Simply I am building a Docker container that will enable me from running my CI/CD script. 
Here is my list of npm commands I need to publish

- Checkout branch.
- Auth with npm regitry
- npm install
- npm run build
- npx semantic-release <- here I want to (publish, generate-notes, change-log) ONLY

> you want this plugin to run npm commands, or you want npm commands to use this plugin to build a container?

**Comment by esatterwhite** [comment]
you probably could if you set `dockerPublish: false` and `dockerClean:false` and tagged them so you know how to reference them.
It's a bit out of scope for what semantic release, and this plugin intend to do. It's probably possible, just not directly supported.

**Comment by tomavic** [comment]
Yeah, my last question is out of scope. But my primary question is the difference between this plugin and the other plugin. 

If you could add the diff. in the README so that anyone can know which one should be used.

I appreciate it

**Comment by esatterwhite** [comment]
@tomavic to me it looks like the package you listed doesn't do image builds or anything to that effect. It just tags and pushes them.
